ibu-imager/seedcleanup/seedcleanup.go

```diff
@@ -0,0 +1,224 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package seedcleanup
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/openshift-kni/lifecycle-agent/ibu-imager/ops"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	// Default location for etcdStaticPodFile
+	etcdStaticPodFile      = "/etc/kubernetes/manifests/etcd-pod.yaml"
+	etcdStaticPodContainer = "etcd"
```

Comment:
added a common package with those constants in #86

Reply:
I didn't mean to imply we should have one package with all the project constants in one place. 

Constants should be in the packages that they are used in, and if they're shared across multiple packages, then they should be made public (first letter uppercase) and placed in the package they're most relevant in. If it doesn't make sense to have the constant in one particular package then it should be moved to a separate common package, and that package doesn't have to be one single universal package for all constants that happen to be shared, it could be still be scoped and we can still have many such common packages.

For now let's leave it as is but consider getting rid of the "utility" package, it's feels unnecessarily generic